packages/cli/src/commands/doctor/doctor.js

```diff
@@ -22,18 +22,20 @@ const printIssue = ({
   label,
   needsToBeFixed,
   isRequired,
+  description,
 }: {
   label: string,
   needsToBeFixed: boolean,
   isRequired: boolean,
+  description: string,
 }) => {
   const symbol = needsToBeFixed
     ? isRequired
       ? chalk.red('✖')
       : chalk.yellow('●')
     : chalk.green('✓');
 
-  logger.log(` ${symbol} ${label}`);
+  logger.log(` ${symbol} ${label}${needsToBeFixed ? ': ' + description : ''}`);
```

Comment:
@jmeistrich @thymikee: what's the reason for adding this?

With this change, this is how it looks:

![image](https://user-images.githubusercontent.com/6207220/64611665-470ae500-d3d2-11e9-9ed1-1fe72a00fb7c.png)

Reply:
That's the reason – to show why the check failed. We treat it as first pass, feel free to iterate on how to present it better :)

Reply:
Perhaps it would be nice to show this only with `--verbose`?

Reply:
Nah, why hiding the reason behind failure? Running doctor again may take some time as well.

Reply:
Good point.

I would like to, at least, get rid of `should be installed` as there isn't much being provided there, thoughts?

Reply:
"is expected to be installed globally" maybe?

Reply:
> is expected to be installed globally

Cool, I like it.

`ios-deploy` is not required to have installed though, the user only needs it if they want to run on the device through the CLI, how would we put that in there?

Reply:
We could set messages per package after running `checkSoftwareInstalled`. This way we can get rid of this a function returning either true or a string. Would you like to refactor?

Reply:
You mean that each health check file can provide its own `description`?

Reply:
Something like this could be added to `getDiagnostics()` output. Currently `needsToBeFixed` can be a boolean or string. But we could make it a bit more explicit and add extra information like error message/descirption. Or we can keep the current behavior and only do something like:
```ts
  getDiagnostics: async () => ({
    needsToBeFixed: await checkSoftwareInstalled('pod') || "custom message",
  }),
```

Reply:
If I got what you mean right, it would be nice to do:

```js
  getDiagnostics: async () => ({
    description: 'required for installing dependencies',
    needsToBeFixed: await checkSoftwareInstalled('pod'),
  }),
```

Is that it?

Reply:
Could be, I'm ok with that.

Reply:
Cool, I'm finishing up some stuff to open a PR, will update #694 with that to refactor it, thanks @thymikee.